### DISCUSSION TITLE
Some questions about the format of meteorological data.

### DISCUSSION THREAD

**Discussion by youzhw-HUST** [mhm-ufz/mHM#81] [Q&A]
Dear developers,
I'm a freshman of mHM, and there are some problems about the format of the meteo data that mHM needs.
I have processed the meteo, morph and other data required by the model according to the format of the sample file, and put them into the specified folder according to the sample requirements. However, when I tried to run the model, the code of the model initialization was reported wrong. After I checked, it was considered that the data format of the nc format weather that may be entered did not match the requirements. My meteorological data nc file is processed in R language with reference to the file format of the given example, but it still cannot be identified by the model. 
So I would like to ask you what kind of method can be quickly downloaded on the network nc format meteorological data (such as ERA5) into the format required for the model, are there any tools or code? I'll be grateful for any help!

**Comment by pallavhydro** [comment]
Hi @youzhw-HUST 

Welcome to mHM.

Could you upload a screenshot each of the following two so that we could help you?

1. The error message during mHM runtime
2. Output of `ncdump -h pre.nc` on your precipitation file.

Cheers
Pallav
